brambling/tests/functional/test_finance_table.py

```diff
@@ -0,0 +1,57 @@
+from django.test import TestCase
+
+from brambling.utils.model_tables import Cell
+from brambling.views.utils import FinanceTable
+from brambling.tests.factories import TransactionFactory, EventFactory, \
+    PersonFactory, OrderFactory
```

Comment:
In keeping with [PEP8](https://www.python.org/dev/peps/pep-0008/#maximum-line-length), we prefer parentheses to backslash. For most cases we indent the continuation line just past the paren on the start line. If there are a *very* large number of items (see urls.py) we instead put each item on a separate line.